xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.h

```diff
@@ -89,6 +89,7 @@ class CRenderBuffer
   AVContentLightMetadata lightMetadata = {};
   std::string stereoMode;
   uint64_t frameIdx = 0;
+  AVColorPrimaries m_originalPrimaries = AVCOL_PRI_BT709;
```

Comment:
Can you move this before  `AVColorPrimaries primaries = AVCOL_PRI_BT709;` ?

(to to have the two related variables together)

Reply:
done